modules/logging-loki-log-access.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-log-access_{context}"]
+= Configuring Loki log access
+In {logging} 5.8 and later versions, you can configure fine grain access to logs using cluster wide policies, as well as by namespace. The ClusterLogging Operator does not grant all users access to logs by default. As an administrator, you may configure your users access as needed.
+
+ClusterRoles grant access to different types of logs:
+
+* `cluster-logging-application-view` grants permissions to read application logs.
+* `cluster-logging-infrastructure-view` grants permissions to read infrastructure logs.
+* `cluster-logging-audit-view` grants permission to read audit logs.
+* `logging-application-logs-reader` - Provides backwards compatibility with prior versions of logging.
+
+If you have upgraded to version 5.8, any authenticated user has permission to read logs in their namespaces. To change this delete the ClusterRole `logging-application-logs-reader` and the ClusterRoleBinding `logging-all-authenticated-application-logs-reader`.
+
+Users who are members of any group specified in the `adminGroups` field of the LokiStack CR are considered admins. Admin users have access to all application logs in all namespaces.
+
+.Example admin group settings
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  tenants:
+    mode: openshift-logging # <1>
+    openshift:
+      adminGroups: # <2>
+      - cluster-admin
+      - custom-admin-group # <3>
+----
+<1> Custom admin groups are only available in this mode.
+<2> Entering an empty list `[]` value for this field disables admin groups.
+<3> Overrides the default groups (`system:cluster-admins`, `cluster-admin`, `dedicated-admin`)
+
+
+ClusterRoleBindings enable authorization cluster wide.
+
+.Example cluster wide log access setting
+[source,yaml]
+----
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: logging-all-application-logs-reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: custom-admin-group # <1>
+subjects: # <2>
+- kind: Group
+  name: system:authenticated
+  apiGroup: rbac.authorization.k8s.io
+----
+<1> Additional `ClusterRoles` are `cluster-logging-infrastructure-view`, and `cluster-logging-audit-view`.
+<2> Specifies the users or groups this object applies to.
+
+
+RoleBindings can be specific to a namespace.
```

Comment:
Is it possible to have some visible distinction between "configure access cluster-wide and configure access per namespace"? Currently this file reads like "first do adminGroups, then do cluster access, then namespaced". I don't think this gets the message across that these are two methods independent from each other and that the `adminGroups` customization is "probably" not needed for most users.